### ISSUE TITLE
Uncomfortable auto pairing

### ISSUE THREAD

**Issue by H16Y** [marktext/marktext#609]
### Description

- [x] Can you reproduce the issue? 

### Steps to reproduce

1. Type at least two "\*". (To get a bold or italic+bold font.)
2. Type a random word.
3. When auto pairing is on, you'll find that you cannot get a perfect pair (by entering more "\*" ). An extra "\*" presents. You'll always have to delete one "\*".

**Expected behavior:**

The pairing should be correct without deleting "\*".

**Actual behavior:**

Described above. It is tolerable but I expect a better user experience.

**Link to an example: [optional]**

### Versions

- Mark Text: v 0.13.65
- Operating system: Win 10 1607 x64

**Comment by mupchrch** [comment]
I think this issue is the same as #739. If so, this one could be closed as well.

**Comment by fxha** [comment]
I'm unsure because v0.13.65 and master have different results and I don't fully understand the issue. @Jocs Can you confirm this is fixed?

**Comment by Jocs** [comment]
@mupchrch Can you explain this issue more exactly.

```md
Type at least two "*". (To get a bold or italic+bold font.)
Type a random word.
When auto pairing is on, you'll find that you cannot get a perfect pair (by entering more "*" ). An extra "*" presents. You'll always have to delete one "*".
```

I can not fully understand this description of above.

**Comment by fxha** [comment]
This issue should be fixed by #739.
